mlir/test/e2e/CMakeLists.txt

```diff
@@ -51,12 +52,17 @@ add_custom_command(OUTPUT ${LOCAL_CONFIG_FILE_BIN}
 )
 
 # Add a command to generate E2E tests for each config file
+# We use a xxx.cfg file associated with xxx.toml file to apply additional guards on the tests.
+# Check conv2d_wrw_perf_config as an example to guard the tests on mfma supported GPUs only.
+# The assumption is that the directory name in xxx.toml is the same as xxx.
 foreach(config ${CONFIGS})
-  set(config_file ${CONFIG_DIR}/${config})
+  set(config_file ${CONFIG_DIR}/${config}.toml)
+  set(lit_cfg_file ${CONFIG_DIR}/${config}.cfg)
   set(config_file_copy ${E2E_DIR}/${config}.copy)
   add_custom_command(OUTPUT ${config_file_copy}
     COMMAND ${CMAKE_COMMAND} -E copy ${config_file} ${config_file_copy}
     COMMAND ${Python3_EXECUTABLE} ${E2E_GEN_SCRIPT} -d ${E2E_DIR} -i ${config_file} ${GEN_MODE}
+    COMMAND if [ -f ${lit_cfg_file} ]\; then cp ${lit_cfg_file} ${E2E_DIR}/${config}/lit.local.cfg\; fi
```

Comment:
I'd put the `${lit_cfg_file}` in quotes in case someone has spaces in their `/path/to/rocmlir`